src/index.js

```diff
@@ -266,12 +266,28 @@ handleOrientation = function () {
   if (device.landscape()) {
     removeClass("portrait");
     addClass("landscape");
+    walkOnChangeOrientationList("landscape");
   } else {
     removeClass("landscape");
     addClass("portrait");
+    walkOnChangeOrientationList("portrait");
   }
 };
 
+changeOrientationList = [];
```

Comment:
Mind declaring this as a var at the top of the file?

Reply:
Declare changeOrientationList as a var at the top